### DISCUSSION TITLE
kobodl seems to not able to contact kobo servers

### DISCUSSION THREAD

**Discussion by steed12** [subdavis/kobo-book-downloader#75] [Q&A] [answered]
Hi,
I installed, configured and used kobodl last week. From yesterdady i am trying to download my entire library again but the script still after the "kobodl book get --get-all". Also with the command for the list, no answer.
With "kobodl user list" i can get my account. So i tried to remove my account and to re add it. All seem to go straight until i enter the captcha code: no answer.
What can be the problem?

Thank you for the answer and the work.

**Comment by subdavis** [comment, marked as answer]
I can confirm that something appears to have changed that breaks kobodl.  I'll investigate soon.  Thanks for the report.

UPDATE: I'm closing this discussion and rolling it over into the issue #76
